### PR TITLE
Add email actions to Environments endpoint.

### PR DESCRIPTION
**Pull request by fiasco**
This change allows support for enable/disable actions for Platform Email on the Environments endpoint.

### CODE REVIEW

File: tests/Endpoints/EnvironmentsTest.php [hunk 1]
```diff
@@ -157,4 +157,32 @@ public function testDeleteCDEnvironment(): void
         $this->assertInstanceOf('\AcquiaCloudApi\Response\OperationResponse', $result);
         $this->assertEquals('The environment is being deleted.', $result->message);
     }
+
+    public testEnableEmail(): void
```
Comment (danepowell) [inline tests/Endpoints/EnvironmentsTest.php:161]:
```suggestion
    public function testEnableEmail(): void
```

File: tests/Endpoints/EnvironmentsTest.php [hunk 2]
```diff
@@ -157,4 +157,32 @@ public function testDeleteCDEnvironment(): void
...
+
+        /** @var \AcquiaCloudApi\Connector\ClientInterface $client */
+        $environment = new Environments($client);
+        $result = $environment->enableEmail('24-a47ac10b-58cc-4372-a567-0e02b2c3d470');
+
+        $this->assertInstanceOf('\AcquiaCloudApi\Response\OperationResponse', $result);
+        $this->assertEquals('Platform Email is being enabled', $result->message);
+    }
+
+    public testDisableEmail(): void
```
Comment (danepowell) [inline tests/Endpoints/EnvironmentsTest.php:175]:
```suggestion
    public function testDisableEmail(): void
```

**Review by danepowell — CHANGES_REQUESTED** [review]
The tests on this project are super easy to run locally: `composer test`

**Review by danepowell — APPROVED** [review]

**Review by danepowell — CHANGES_REQUESTED** [review]
Looks like you've still got failing tests. Let me know if you're having problems running them locally.

**Comment by fiasco** [comment]
I tried out github's IDE but it doesn't provide an executable environment...

**Comment by danepowell** [comment]
> I tried out github's IDE but it doesn't provide an executable environment...

That doesn't sound right. I just created a codespace, immediately saw a terminal window and typed `composer install && composer test`, everything worked great. Well, except for the failing tests :)

**Comment by danepowell** [comment]
Sorry, looks like you got bit by #317. I updated your PR so tests will at least run and pass, but we'll need for #317 to be merged first.

**Comment by danepowell** [comment]
I have no clue why mutation tests are failing now. Some wires have really gotten crossed somewhere because they claim composer.lock still has a reference to phpunit 9.6.3, which it certainly does not. Possibly because you are using `master` instead of a feature branch? 😕

**Comment by danepowell** [comment]
Sorry, now blocked on https://github.com/codecov/codecov-action/issues/837

**Pull request merged**